sql-testingoutput.sql

```diff
@@ -16,6 +16,8 @@
 
 SELECT votes FROM articles;
 
-SELECT votes + 1 FROM articles WHERE article_id = 1;
+-- SELECT votes + 10 AS updated_votes FROM articles WHERE article_id = 9;
 
--- UPDATE articles SET votes + 1 WHERE article_id = 1 RETURNING *;
\ No newline at end of file
+UPDATE articles SET votes = votes + 10 WHERE article_id = 9;
```

Comment:
This file shouldn't be included in your PR, I assume it's being used to play around with SQL